### PR TITLE
Hide input search when data array is empty

### LINKED ISSUE #7: Hide input search when data array is empty
## Description
Hide input search when data array is empty #6

### PR DESCRIPTION
**Pull request by totanvix**
Hi Worksout,

I got this error when I typing search input with data array empty.

image

That is reason for this PR.

Thanks for your reading.

### CODE REVIEW

**Comment by owonie** [comment]
I'll check this soon. Thank you for contributing :)

**Comment by owonie** [comment]
@totanvix  I found that new code will disable renderSearchView when data (data array) is not empty. I will make a new issue for this, if you have a idea, just tell me  :)

**Comment by totanvix** [comment]
> @totanvix I found that new code will disable renderSearchView when data (data array) is not empty. I will make a new issue for this, if you have any idea, just tell me :)

@owonie I just updated my code to fix what you say, you can check it.

File: src/SelectDropdown.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ const SelectDropdown = (
   /* ******************** Render Methods ******************** */
   const renderSearchView = () => {
     return (
-      search && (
+      search && data.length && (
```
Comment (owonie) [inline src/SelectDropdown.js:137]: Remove this line and keep 230 line codes would be more stable.
If keep 137 line, user can't do searching again when search results is empty :)

**Review by owonie — CHANGES_REQUESTED** [review]

File: src/SelectDropdown.js (hunk 1 above)
Comment (totanvix) [reply src/SelectDropdown.js:137]: I use `data.length`, not `dataArr.length`. And it's still possible for users search again.

Comment (totanvix) [reply src/SelectDropdown.js:137]: At the first time in this PR, I used `dataArr.length` at line 137, that might confuse you this time.

**Pull request merged**